deployments/README.md

```diff
@@ -1,77 +1,89 @@
-  - `ETHEREUM_HASH` - commit hash within that repo (can also be a branch or tag)
-  - `PROJECT` - a project to build withing bridges repo (`rialto-bridge-node` or
-    `ethereum-poa-relay` currently)
+```bash
+docker-compose -f ./networks/docker-compose.rialto.yml up
+```
 
-  You can however uncomment `ADD` commands within the docker files to build
-  an image from your local sources.
+After running this command we would have a network of several nodes producing blocks.
 
-### Requirements
+## Bridges
+A _bridge_ is a way for several _networks_ to connect to one another. Bridge deployments have their
+own Docker Compose files which can be found in the `./bridges` folder. These Compose files typically
+contain bridge relayers, which are services external to blockchain nodes, and other components such
+as testing infrastructure, or front-end code.
 
-Make sure to install `docker` and `docker-compose` to be able to run & test
-bridges deployments locally.
+Unlike the network Compose files, these *cannot* be deployed on their own. They must be combined
+with different networks.
 
-### Polkadot.js UI
+In general, we can deploy the bridge using `docker-compose up` in the following way:
 
-To teach the UI decode our custom types used in the pallet, go to: `Settings -> Developer`
-and import the [`./types.json`](./types.json)
+```bash
+docker-compose -f <bridge>.yml \
+               -f <network_1>.yml \
+               -f <network_2>.yml \
+               -f <monitoring>.yml up
+```
+
+If you want to see how the Compose commands are actually run, check out the source code of the
+[`./run.sh`](./run.sh).
 
-## Rialto
+One thing worth noting is that we have a _monitoring_ Compose file. This adds support for Prometheus
+and Grafana. We cover these in more details in the [Monitoring](#monitoring) section. At the moment
+the monitoring Compose file is _not_ optional, and must be included for bridge deployments.
 
-`Rialto` is a test bridge network deployment between a test Ethereum PoA network and
-test Substrate network.
-Its main purpose is to make sure that basic PoA<>Substrate bridge operation works.
-The network is being reset every now and then without a warning.
+### Running and Updating Deployments
+We currently support two bridge deployments
+1. Ethereum PoA to Rialto Substrate
+2. Rialto Substrate to Millau Substrate
 
-### Docker-Compose Deployment
+These networks can be deployed using our [`./run.sh`](./run.sh) script. We need to call the script
+from the root of the `deployments` folder since it assumes that for some paths.
 
-To run a full network with two-way bridge functionality and cross-chain transfers you
-may use the `docker-compose.yml` file in the [`rialto`](./rialto) folder. This will pull
-images from the Docker Hub for all the components.
+The first argument it takes is the name of the bridge you want to run. Right now we only support two
+networks: `eth-poa-sub` and `rialto-millau`.
```

Comment:
```suggestion
networks: `poa-rialto` and `rialto-millau`.
```

IMHO it should be called `poa-rialto` not `eth-poa-sub`. `poa` being just a codename for the `ETH PoA` test network. The name is already used like that in services definitions (e.g. `poa-node-arthur` and `rialto-node-alice`)